From 20a0a211cff993413c6d5cf431d0608d147f5a05 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Mon, 13 Jan 2020 11:18:22 -0600
Subject: [PATCH 45/45] arm64: dts: logicpd-imx8mm-kit: Remove modem reset node

The modem reset node isn't being referenced from anyone, and
the Bluetooth driver expliticly has a gpio reference for pulling
the device out of reset.

This patch removes the un-needed modem reset node.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
index f0929c06ca65..ac8a682a9037 100644
--- a/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
@@ -31,13 +31,6 @@
 		};
 	};
 
-	modem_reset: modem-reset {
-		compatible = "gpio-reset";
-		reset-delay-us = <2000>;
-		reset-post-delay-ms = <40>;
-		#reset-cells = <0>;
-	};
-
 	reg_pcie_pwr_en: regulator-pci_pwr_en {
 		compatible = "regulator-fixed";
 		regulator-name = "pci_pwr_en";
-- 
2.17.1

